src/js/tabs/configuration.js

```diff
@@ -311,6 +311,27 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // translate to user-selected language
         i18n.localizePage();
 
+        var gyro_align_content_e = $('.tab-configuration .gyro_align_content');
+        var legacy_gyro_alignment_e = $('.tab-configuration .legacy_gyro_alignment');
+        var legacy_accel_alignment_e = $('.tab-configuration .legacy_accel_alignment');
+
+        // Hide the new multi gyro element by default
+        gyro_align_content_e.hide();
+
+        // If we have the newer MSP API, and USE_MULTI_GYRO is defined in the target, show the new element, while hiding the old ones.
+        if (semver.gte(CONFIG.apiVersion, "1.41.0") &&
+            SENSOR_ALIGNMENT.use_multi_gyro == 1) {
+            gyro_align_content_e.show();
+            legacy_gyro_alignment_e.hide();
+            legacy_accel_alignment_e.hide();
+        }
```

Comment:
I think is better to remove the validation of the version here. The `use_multi_gyro` has a value of zero by default for older versions and it will let a cleaner code.